packages/components/src/components/hds/dropdown/index.ts

```diff
@@ -137,7 +144,7 @@ export default class HdsDropdown extends Component<HdsDropdownSignature> {
     classes.push(`hds-dropdown__content--position-${this.listPosition}`);
 
     // add a class based on the @width argument
-    if (this.args.width) {
+    if (this.args.width || this.args.matchToggleWidth) {
       classes.push('hds-dropdown__content--fixed-width');
```

Comment:
[question] in the template we're setting `{{style width=@width max-height=@height}}` independently from the fact that `@matchToggleWidth` is true or false; should we instead avoid assigning a `width` if the value is true?

(practically it doesn't change, because _I think_ what is currently happening is that it sets an initial width equal to the `@width` value but then the Popover API takes over and assigns the toggle width value.

No strong opinions, just wanted to gather your thought on what is better to do in this case.

Reply:
> (practically it doesn't change, because I think what is currently happening is that it sets an initial width equal to the @width value but then the Popover API takes over and assigns the toggle width value.

yes, that's how it behaves. I gave this a thought but wasn't sure it was worth interfering with the style modifier to avoid an override